modules/new_profile/jsx/NewProfileForm.js

```diff
@@ -36,8 +34,6 @@ class NewProfileForm extends React.Component {
                 method: 'GET',
                 dataType: 'json',
                 success: (data) => {
-                    // FIXME: Remove the following line of code as soon as hiddenHeaders is
-                    // accepted as a prop by the StaticDataTable Component.
```

Comment:
shouldn't the code be removed too instead of just the comment saying to remove it?